### PR TITLE
Feature/account product fiscal classification migration 8.0

### PR DESCRIPTION
**Pull request by renatonlima**

### CODE REVIEW

File: account_product_fiscal_classification/account_product_fiscal_classification.py [hunk 1]
```diff
@@ -20,187 +20,145 @@
...
-                    domain=[('type_tax_use', '!=', 'sale')])
-    }
-
-    def button_update_products(self, cr, uid, ids, context=None):
+
+    name = fields.Char('Main code', size=32, required=True)
+    description = fields.Char('Description', size=64)
+    company_id = fields.Many2one('res.company', 'Company')
+     # TODO restrict to company_id if company_id set when
+     # framework will allow it:
```
Comment (bwrsandman) [inline account_product_fiscal_classification/account_product_fiscal_classification.py:34]: Does the framework allow it now?

File: account_product_fiscal_classification/account_product_fiscal_classification.py [hunk 2]
```diff
@@ -20,187 +20,145 @@
...
+                recs = self.search([('name', operator, name)] + args,
+                    limit=limit)
+                recs += self.search([('description', operator, name)] + args,
+                    limit=limit)
+        if not recs:
+            recs = self.search([('name', operator, name)] + args, limit=limit)
+        return recs.name_get()
+
+
+class AccountProductFiscalClassificationTemplate(models.Model):
```
Comment (bwrsandman) [inline account_product_fiscal_classification/account_product_fiscal_classification.py:97]: Can you put this class in its own script `account_product_fiscal_classification_template.py`

File: account_product_fiscal_classification/account_product_fiscal_classification.py [hunk 3]
```diff
@@ -20,187 +20,145 @@
...
-        if not args:
-            args = []
-
-        if not context:
-            context = {}
 
+    name = fields.Char('Main code', size=32, required=True)
+    description = fields.Char('Description', size=64)
+    # TODO restrict to company_id if company_id set when
+    # framework will allow it:
```
Comment (bwrsandman) [inline account_product_fiscal_classification/account_product_fiscal_classification.py:104]: Does it allow it?

File: account_product_fiscal_classification/account_product_fiscal_classification.py [hunk 4]
```diff
@@ -20,187 +20,145 @@
...
+                recs = self.search([('name', operator, name)] + args,
+                    limit=limit)
+                recs += self.search([('description', operator, name)] + args,
+                    limit=limit)
+        if not recs:
+            recs = self.search([('name', operator, name)] + args, limit=limit)
+        return recs.name_get()
+
+
+class WizardAccountProductFiscalClassification(models.TransientModel):
```
Comment (bwrsandman) [inline account_product_fiscal_classification/account_product_fiscal_classification.py:135]: Can you put this class in its own script `wizard/wizard_account_product_fiscal_classification.py`?

File: account_product_fiscal_classification/account_product_fiscal_classification_data.xml [hunk 5]
```diff
@@ -1,16 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
```
Comment (bwrsandman) [inline account_product_fiscal_classification/account_product_fiscal_classification_data.xml:1]: Please remove this file.

File: account_product_fiscal_classification/product.py [hunk 6]
```diff
@@ -20,33 +20,29 @@
 #
 ###############################################################################
 
-from osv import fields, osv
+from openerp import models, fields, api
 
 
-class product_template(osv.Model):
+class ProductTemplate(models.Model):
```
Comment (bwrsandman) [inline account_product_fiscal_classification/product.py:26]: Please put this class in its own script `product_template.py`

**Comment by gurneyalex** [comment]
@renatonlima please resubmit on branch 8.0

**Pull request closed without merging**